src/Symfony/Bundle/TwigBundle/Loader/FilesystemLoader.php

```diff
@@ -89,7 +89,14 @@ protected function findTemplate($template, $throw = true)
         }
 
         if (false === $file || null === $file) {
-            throw new \Twig_Error_Loader(sprintf('Unable to find template "%s".', $logicalName), -1, null, $previous);
+            try {
+                list($namespace, $name) = $this->parseName($logicalName);
```

Comment:
I see. In your opinion, what would be the easiest way to overcome this issue then?